packages/preferences-controller/src/PreferencesController.ts

```diff
@@ -299,6 +301,15 @@ export class PreferencesController extends BaseController<
   setShowTestNetworks(showTestNetworks: boolean) {
     this.update({ showTestNetworks });
   }
+
+  /**
+   * A setter for the user enable the IPFS gateway
```

Comment:
Nit: The fact that this method enables the IPFS gateway is evident from the name of the method itself. It might be more helpful for those reading the code to explain, what effect does this property have on the greater system? What does it mean to enable the IPFS gateway?